### PR TITLE
Cucumber

### PR DESCRIPTION
**Pull request by jaffamonkey**
Important test updates (i.e. some tests were failing incorrectly)

### CODE REVIEW

File: tests/wdio.BROWSERSTACK.conf.js [hunk 1]
```diff
@@ -6,14 +6,18 @@ config.capabilities = [{
...
+        debug: true
 }];
 
 config.services = ['browserstack'];
 config.user = 'euniceaidoo1';
 config.key = 'z8aabpqRzKvWZDHKTKff';
 config.browserstackLocal = true;
 config.host = 'hub-cloud.browserstack.com';
-config.baseUrl = 'https://localhost:80';
+config.baseUrl = 'https://par-beta-test.cloudapps.digital';
```
Comment (kalpaitch) [inline tests/wdio.BROWSERSTACK.conf.js]: @jaffamonkey Is this intentional? Pointing it at our live site?

Comment (jaffamonkey) [inline tests/wdio.BROWSERSTACK.conf.js]: That is intentional - more pragmatic as tests commonly fail testing localhost urls, down to severe lag time.

Comment (jaffamonkey) [inline tests/wdio.BROWSERSTACK.conf.js]: This only affects cucumber js run against browserstack service

File: tests/wdio.BROWSERSTACK.conf.js [hunk 2]
```diff
@@ -6,14 +6,18 @@ config.capabilities = [{
...
         os: 'Windows',
         os_version: '7',
-        resolution: '1024x768'
+        resolution: '1024x768',
+        debug: true
 }];
 
 config.services = ['browserstack'];
 config.user = 'euniceaidoo1';
 config.key = 'z8aabpqRzKvWZDHKTKff';
```
Comment (kalpaitch) [inline tests/wdio.BROWSERSTACK.conf.js]: Also, is this a live access key?? If so we'll need to remove from public repo and revoke key??

**Pull request merged**